### ISSUE TITLE
[Feature request] autostart script

### ISSUE THREAD

**Issue by bonnebulle** [jlesage/docker-jdownloader-2#218]
### Idea

Add a env variable like "STARTUP_SRIPT", lanching before jdownloader startup script...
I try using Openrc ( but cant make my openvpn works )

**Comment by jlesage** [comment]
One way to achieve this to map your script to `/etc/cont-init.d/` in the container.

**Comment by jlesage** [comment]
> There is no 'in app' (GUI) option ?

What do you mean exactly ? I don't think JDownloader itself can start a script.

**Comment by bonnebulle** [comment]
yes, it was my point.... do JD gui include starting up script/action !
Thanks again
> /etc/cont-init.d/ 

do the job very well !
